### PR TITLE
winpr: fix unitary test when CreateProcess is CreateProcessW

### PR DESCRIPTION
**Pull request by hardening**
As the second argument of CreateProcessW is modified to split with spaces and so this can not be a const arg.

### CODE REVIEW

File: winpr/include/winpr/crt.h [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <malloc.h>
```
Comment (akallabeth) [inline winpr/include/winpr/crt.h:26]: non standard header, should already be included with `stdlib.h`, breaks build on CI

**Review by akallabeth — CHANGES_REQUESTED** [review]
Overall LGTM, but you need to get rid of the header

File: winpr/libwinpr/thread/test/TestThreadCreateProcess.c [hunk 2]
```diff
@@ -15,7 +15,14 @@ int TestThreadCreateProcess(int argc, char* argv[])
 	BOOL status;
 	DWORD exitCode;
 	LPCTSTR lpApplicationName;
-	LPTSTR lpCommandLine;
+
+#ifdef _WIN32
+	TCHAR lpCommandLine[200] = _T("cmd /C set");
```
Comment (akallabeth) [inline winpr/libwinpr/thread/test/TestThreadCreateProcess.c:20]: Sure this will copy the string to the buffer and not just set a pointer?

Comment (hardening) [reply winpr/libwinpr/thread/test/TestThreadCreateProcess.c:20]: I think so, at least the unit test passes correctly.

**Comment by hardening** [comment]
Changes done

**Review by akallabeth — APPROVED** [review]

**Pull request merged**